### PR TITLE
Fixes to common code

### PR DESCRIPTION
**Pull request by Cerfoglg**
Quick fixes to resolve some issues

Should also fix https://travis-ci.org/benchflow/collectors/builds/138088460
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/benchflow/commons/pull/11%23discussion_r67343288%22%2C%20%22https%3A//github.com/benchflow/commons/pull/11%23discussion_r67386925%22%2C%20%22https%3A//github.com/benchflow/commons/pull/11%23discussion_r67388694%22%5D%2C%20%22comments%22%3A%20%7B%22Pull%20dc17b8b4018df44366ea9270afee19cb75645d43%20minio/go/src/cloud/benchflow/commons/minio/minio.go%2023%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/benchflow/commons/pull/11%23discussion_r67343288%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22The%20port%20should%20be%20a%20parameter%22%2C%20%22created_at%22%3A%20%222016-06-16T13%3A28%3A01Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7163641%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/VincenzoFerme%22%7D%7D%2C%20%7B%22body%22%3A%20%22%40VincenzoFerme%20Added%22%2C%20%22created_at%22%3A%20%222016-06-16T17%3A20%3A44Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/8819678%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Cerfoglg%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222016-06-16T17%3A31%3A26Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7163641%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/VincenzoFerme%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20minio/go/src/cloud/benchflow/commons/minio/minio.go%3AL30-42%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [x] <a href='#crh-comment-Pull dc17b8b4018df44366ea9270afee19cb75645d43 minio/go/src/cloud/benchflow/commons/minio/minio.go 23'></a> &nbsp;<b><a href='https://github.com/benchflow/commons/pull/11#discussion_r67343288'>File: minio/go/src/cloud/benchflow/commons/minio/minio.go:L30-42</a></b>
- <a href='https://github.com/VincenzoFerme'></a> The port should be a parameter
- <a href='https://github.com/Cerfoglg'></a> @VincenzoFerme Added

<a href='https://www.codereviewhub.com/benchflow/commons/pull/11?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/benchflow/commons/pull/11?approve=1'></a>&nbsp;<a href='https://github.com/benchflow/commons/pull/11'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by VincenzoFerme** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/benchflow/commons/pull/11'>Refresh</a>.<a href='#crh-update-none'></a>

File: minio/go/src/cloud/benchflow/commons/minio/minio.go [hunk 1]
```diff
@@ -34,12 +30,12 @@ func callMinioClient(fileName string, minioHost string, minioKey string) {
...
 */
 
-func sendGzipToMinio(fileName string, minioHost string, minioKey string, accessKey string, secretAccessKey string) {
+func SendGzipToMinio(fileName string, minioHost string, minioKey string, accessKey string, secretAccessKey string) {
 	sendToMinio(fileName, minioHost, minioKey, accessKey, secretAccessKey, "application/gzip")
 	}
 
 func sendToMinio(fileName string, minioHost string, minioKey string, accessKey string, secretAccessKey string, fileType string) error {
-	minioClient, err := minio.New(minioHost, accessKey, secretAccessKey, true)
+	minioClient, err := minio.New(minioHost+":9000", accessKey, secretAccessKey, false)
```
Comment (VincenzoFerme) [inline minio/go/src/cloud/benchflow/commons/minio/minio.go]: The port should be a parameter, as well as the `false`.

Comment (Cerfoglg) [inline minio/go/src/cloud/benchflow/commons/minio/minio.go]: @VincenzoFerme Added

Comment (VincenzoFerme) [inline minio/go/src/cloud/benchflow/commons/minio/minio.go]: :+1:

**Pull request merged**